src/ProgressOnderwijsUtils/SchemaReflection/DatabaseDefinitionScripter.cs

```diff
@@ -24,7 +24,7 @@ public void TableDefinitionScript(StringBuilder sb, DatabaseDescription.Table ta
                     + (colMetaData.IsPrimaryKey ? "PK;" : "")
                     + "colId:"
                     + colMetaData.ColumnId;
-                _ = sb.Append("    " + separatorFromPreviousCol + colMetaData.ColumnName + " as " + SqlServerUtils.PrettifySqlExpression(definition.Definition) + persistedClause + columnTrivia + "\n");
+                _ = sb.Append("    " + separatorFromPreviousCol + colMetaData.ColumnName + " as " + SqlServerUtils.PrettifySqlExpression(definition.Definition) + collationClause + persistedClause + columnTrivia + "\n");
```

Comment:
In fd7b0f837a035365ea7f7c9de9befe77af516ec5 doe je moeite om de collation voor de nullability te plaatsen; dat hoeft hier niet?

Reply:
omdat het een computed column is, gok ik?

Reply:
Normaal gesproken moet de collation voor de nullability staan, anders krijg je een sql-syntax-error. Dus al je met dit script ooit ook de database zou willen kunnen aanmaken is dat wel nodig lijkt me.

Reply:
De nullability zit in de `persistedClause` en de collation staat dus daarvoor. Ik heb nu ook het zetten van de collation voor het zetten van de persisted/nullability gezet.